src/lightning/fabric/plugins/precision/deepspeed.py

```diff
@@ -88,7 +85,7 @@ def convert_input(self, data: Any) -> Any:
     def convert_output(self, data: Any) -> Any:
         return apply_to_collection(data, function=_convert_fp_tensor, dtype=Tensor, dst_type=torch.get_default_dtype())
 
-    def backward(self, tensor: Tensor, model: "deepspeed.DeepSpeedEngine", *args: Any, **kwargs: Any) -> None:
+    def backward(self, tensor: Tensor, model: "DeepSpeedEngine", *args: Any, **kwargs: Any) -> None:
```

Comment:
Why not keep the class package?

Reply:
Just to get rid of the squiggly lines in PyCharm. The import and type hint is still there and correct.